### PR TITLE
Fg/Update docs for self-hosted peps

### PR DESCRIPTION
**Pull request by fernando-guerrero-leon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,3 +112,15 @@ EXAMPLES:
...
+# WIREGUARD PROTOCOL
+
+Optionally support for the WIREGUARD protocol can be enabled by installing the 
+wireguard kernel module on the target host.
+
+```
+sudo yum install epel-release https://www.elrepo.org/elrepo-release-7.el7.elrepo.noarch.rpm
+sudo yum install yum-plugin-elrepo
+sudo yum install kmod-wireguard
+```
```
Comment (dmibaranov) [inline README.md]: please add a link to official docs https://www.wireguard.com/install/

File: go.sh [hunk 2]
```diff
@@ -429,6 +429,8 @@ esac
             || fail "Failed to pull WatchTower image '$wt_image' from Docker Hub"
     }
 
+    modinfo wireguards &> /dev/null || rem "Missed WireGuard kernel module"
```
Comment (dmibaranov) [inline go.sh]: /WireGuard/wireguard/ and please add ", protocol is disabled"

Comment (fernando-guerrero-leon) [reply go.sh]: Done!

File: README.md (hunk 1 above)
Comment (fernando-guerrero-leon) [reply README.md]: Done!

**Review by dmibaranov — APPROVED** [review]
thanks!

**Pull request merged**